### PR TITLE
[android] Run Android Studio code formatter for all .gradle files

### PR DESCRIPTION
**Pull request by sjchmiela**
# Why

To make https://github.com/expo/expo/pull/2446 easier to review and cause it's nice to have files formatted properly.

# How

Iterated manually through every Gradle file visible in the left pane of Android Studio and for each ran <kbd>Reformat Code</kbd> command.

# Test Plan

Project synced without problems.

### CODE REVIEW

File: android/versioned-abis/expoview-abi25_0_0/build.gradle [hunk 1]
```diff
@@ -89,7 +89,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi25_0_0/build.gradle]: move this comment above `api 'com.amplitude:android-sdk:2.9.2'`

File: android/versioned-abis/expoview-abi26_0_0/build.gradle [hunk 2]
```diff
@@ -89,7 +89,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi26_0_0/build.gradle]: move this comment

File: android/versioned-abis/expoview-abi27_0_0/build.gradle [hunk 3]
```diff
@@ -89,7 +89,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi27_0_0/build.gradle]: move this comment

File: android/versioned-abis/expoview-abi28_0_0/build.gradle [hunk 4]
```diff
@@ -89,7 +89,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi28_0_0/build.gradle]: move this comment

File: android/versioned-abis/expoview-abi29_0_0/build.gradle [hunk 5]
```diff
@@ -106,7 +106,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi29_0_0/build.gradle]: move this comment

File: android/versioned-abis/expoview-abi30_0_0/build.gradle [hunk 6]
```diff
@@ -118,7 +118,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi30_0_0/build.gradle]: move this comment

File: android/versioned-abis/expoview-abi31_0_0/build.gradle [hunk 7]
```diff
@@ -90,7 +90,8 @@ dependencies {
   }
   compileOnly 'org.glassfish:javax.annotation:3.1.1'
   api 'de.greenrobot:eventbus:2.4.0'
-  api 'com.amplitude:android-sdk:2.9.2' // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
+  api 'com.amplitude:android-sdk:2.9.2'
+  // Be careful when upgrading! Upgrading might break experience scoping. Check with Jesse. See Analytics.resetAmplitudeDatabaseHelper
```
Comment (tsapeta) [inline android/versioned-abis/expoview-abi31_0_0/build.gradle]: move this comment

**Review by tsapeta — APPROVED** [review]

**Review by bbarthec — CHANGES_REQUESTED** [review]
What about `packages/expo-module-template/android/build.gradle`?

**Review by bbarthec — APPROVED** [review]

**Pull request merged**